regression/cbmc/CMakeLists.txt

```diff
@@ -1,3 +1,3 @@
 add_test_pl_tests(
-    "$<TARGET_FILE:cbmc>"
+    "$<TARGET_FILE:cbmc> -X smt-backend"
```

Comment:
I'm afraid this will need more work as it passes that single string as one option to test.pl. Some hack to add_test_pl_tests will be required to add that capability. It might actually suffice to remove the quotes around `${cmdline}` in its macro definition in regression/CMakeLists.txt.